lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -249,7 +249,7 @@ def add_payment_sender_data(post, options)
         post[:sender][:type] = sender[:type] if sender[:type]
         post[:sender][:first_name] = sender[:first_name] if sender[:first_name]
         post[:sender][:last_name] = sender[:last_name] if sender[:last_name]
-        post[:sender][:dob] = sender[:dob] if sender[:dob]
+        post[:sender][:date_of_birth] = sender[:date_of_birth] if sender[:date_of_birth]
```

Comment:
```suggestion
        post[:sender][:date_of_birth] = birthday if birthday
```

Reply:
This would also require a `birthday = sender[:dob] || sender[:date_of_birth]` added above, but I can't make suggestions up that high